### PR TITLE
Feature/integrate with linked in

### PR DESCRIPTION
**Pull request by bulgakovk**
LinkedIn will redirect you from [auth link](https://www.linkedin.com/oauth/v2/authorization?&response_type=code&client_id=77g8m9xwl0j0a8&redirect_uri=http://127.0.0.1:9000/linkedIn/auth&state=1&scope=r_basicprofile) to this route.
ToDo:
1) Check "state" code.
2) Do something with response body.

### CODE REVIEW

File: server/routes.js [hunk 1]
```diff
@@ -18,6 +19,64 @@ export default function(app) {
   app.route('/:url(api|auth|components|app|bower_components|assets)/*')
    .get(errors[404]);
 
+  app.get('/linkedIn/auth', (req, res) => {
+    if (req.query.error) {
+      /*
+      Need to handle event: "The user cancelled LinkedIn login". What should we do?
```
Comment (kononenko-a-m) [inline server/routes.js]: Error should be sent to client,  HTTP code 500 + response like  {error: 'linkendinAuthCanceled'}

File: server/routes.js [hunk 2]
```diff
@@ -18,6 +19,64 @@ export default function(app) {
...
+      */
+      res.send(req.query.error_description);
+      return;
+    }
+
+    request.post({url : 'https://www.linkedin.com/oauth/v2/accessToken',
```
Comment (kononenko-a-m) [inline server/routes.js]: is this route would be called from linkendin?

Comment (bulgakovk) [inline server/routes.js]: Yup. Linkedin automaticly redirects to this url after succesful authentication if we're talking about 
>/linkedIn/auth

**Comment by itmozok-kiev** [comment]
This registration made without Passport.js. So I closed this pull-request.

**Pull request closed without merging**